src/scss/_index_sections.scss

```diff
@@ -158,6 +158,104 @@
       margin: 50px auto;
     }
   }
+
+  @keyframes spongieDance {
+    0% {
+      transform: translate(0, 0) rotate(-7deg);
+    }
+    25% {
+      transform: translate(0, -15px) rotate(0deg);
+    }
+    50% {
+      transform: translate(0, 0) rotate(7deg);
+    }
+    75% {
+      transform: translate(0, -15px) rotate(0deg);
+    }
+    100% {
+      transform: translate(0, 0) rotate(-7deg);
+    }
+  }
+
+  @-webkit-keyframes spongieDance {
```

Comment:
This is not needed, [autoprefixer](https://github.com/postcss/autoprefixer) will automatically handle adding the necessary prefixes.